create_rental.rb

```diff
@@ -0,0 +1,25 @@
+require './rental'
+
+module CreateRental
+  def create_rental
+    puts 'Select book by ID: '
+    @books.each_with_index do |book, index|
+      puts "ID: #{index + 1}, Title: #{book.title}, Author: #{book.author}"
+    end
+    book_id = gets.chomp.to_i
+    puts 'Book selected'
+
+    puts 'Select person by ID: '
+    @people.each_with_index do |person, index|
+      puts "ID:#{index + 1}, Name: #{person.name}, Age: #{person.age}"
+    end
+    person_id = gets.chomp.to_i
+    puts 'Person selected'
+
+    print 'Date (yyyy-mm-dd): '
+    date = gets.chomp.to_s
+    puts 'Date of renting book is added'
+    @rentals << Rental.new(date, @people[person_id - 1], @books[book_id - 1])
```

Comment:
- [ ] Kindly notice that you are swapping the arguments when you are creating the `rental` and this causing error please fix that.

```ruby
@rentals << Rental.new(date, @books[book_id - 1], @people[person_id - 1])
```
![Capture](https://user-images.githubusercontent.com/33752735/200601344-cea47255-70ea-4ad5-b6bd-667d03623c58.PNG)

Reply:
Thank you @ShahierNashaat I couldn't find the source of the error and I submitted in the hopes that a reviewer would see it and show me the fix lol